builder/frameworks/arduino.py

```
@@ -28,250 +28,875 @@
+    # Windows CMD has ~32768 character limit, we use aggressive values close to this
+    platform_defaults = {
+        "esp32": 35000,      # Standard ESP32
+        "esp32s2": 35000,    # ESP32-S2
+        "esp32s3": 35000,    # ESP32-S3
+        "esp32c3": 33000,    # ESP32-C3
+        "esp32c2": 33000,    # ESP32-C2
+        "esp32c6": 32500,    # ESP32-C6
+        "esp32h2": 35000,    # ESP32-H2
+        "esp32p4": 35000,    # ESP32-P4
+    }
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)